src/shipit_code_coverage/tests/test_report_generators.py

```diff
@@ -28,10 +85,18 @@ def test_zero_coverage(tmpdir,
         zero_coverage_functions = json.load(f)
 
     expected_zero_coverage_functions = [
-        {'funcs': 1, 'name': 'mozglue/build/dummy.cpp', 'uncovered': True},
-        {'funcs': 2, 'name': 'toolkit/components/osfile/osfile.jsm', 'uncovered': False},
-        {'funcs': 1, 'name': 'js/src/jit/JIT.cpp', 'uncovered': False},
-        {'funcs': 1, 'name': 'toolkit/components/osfile/osfile-win.jsm', 'uncovered': True},
+        {'funcs': 1, 'name': 'mozglue/build/dummy.cpp', 'uncovered': True,
+         'size': 1, 'commits': 2,
+         'first_push_date': '', 'last_push_date': ''},
```

Comment:
Why are all the dates empty? Can you make at least one non-emtpy, so we actually make sure that the dates are working?

Reply:
the dates depends on when the test is ran or maybe we can hack the sqlite table to put the dates we want, wdyt ?

Reply:
I guess using the dates of when the test is run should be fine (and easier and less prone to regressions than hacking the sqlite table)